deepfence_ui/app/scripts/components/topology-view/donut-details-modal/donut-details-modal.js

```diff
@@ -164,6 +169,19 @@ class DonutDetailsModal extends React.Component {
     }, 0);
   }
 
+  onSort(data) {
+    if(data[0]?.desc === true) {
+      this.setState({ sortSeverity: 'desc'})
+    } else if (data[0]?.desc === false) {
+      this.setState({sortSeverity: 'asc'})
+    } else {
+      this.setState({ sortSeverity: 'desc'})
+    }
+    setTimeout(() => {
+      this.fetchAlertsDetails();
+    }, 0);
```

Comment:
we need to check which column is getting sorted here. and apply sorting only if severity column is being sorted. the logic to set state and trigger api call can be made better with something like

```suggestion
    this.setState({
      sortSeverity: !data[0]?.desc ? 'asc' : 'desc' 
    }, () => {
      this.fetchAlertsDetails();
    });
```

Reply:
We should enable sorting for only severity column, and disable it on all other columns. having `enableSorting` true on table means all the columns will be sortable.

Reply:
@manV  right. 
I will do the changes. Thanks.